modules/web/themes/halo/samples/components/icons/icons.scss

```diff
@@ -13,7 +21,16 @@
     height: inherit;
   }
 
-  .v-slot-sampler-scroll .v-slot-c-scrollbox {
+  .v-slot-sampler-scroll {
     height: inherit;
   }
+
+  .vbox-icon:hover {
```

Comment:
Add paddings to the `vbox-icon`. it seems that `top: 10px` and `bottom 14px` look good.